nsx_vtep/actions/nsxraml/html-version/readme.md

```diff
@@ -0,0 +1,19 @@
+# HTML Version of the NSX RAML Specification
```

Comment:
We are using nsxraml (https://github.com/vmware/nsxraml) for NSX REST APIs.
Right now some of the API are being used in the pack and rest can be needed in future enhancement.